android-core/src/main/java/com/mparticle/modernization/core/MParticleMediator.kt

```diff
@@ -0,0 +1,53 @@
+package com.mparticle.modernization.core
+
+import com.mparticle.MParticleOptions
+import com.mparticle.modernization.eventlogging.MParticleEventLogging
+import com.mparticle.modernization.eventlogging.example.MParticleEventLoggingImpl
+import com.mparticle.modernization.identity.InternalIdentity
+import com.mparticle.modernization.identity.example.MParticleIdentityImpl
+import com.mparticle.modernization.kit.KitManagerInternal
+import com.mparticle.modernization.kit.MParticleKit
+import com.mparticle.modernization.kit.MParticleKitManagerImpl
+import com.mparticle.modernization.kit.example.MpKit
+import com.mparticle.modernization.uploading.MParticleDataUploader
+import com.mparticle.modernization.uploading.MParticleDataUploaderImpl
+import com.mparticle.modernization.uploading.MParticleUploadingStrategy
+import kotlinx.coroutines.CloseableCoroutineDispatcher
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.SupervisorJob
+import kotlinx.coroutines.asCoroutineDispatcher
+import java.util.concurrent.Executors
+
+class MParticleMediator {
```

Comment:
Can you give us a brief overview of what is the purpose of the MParticleMediator? Is it analogous to the mp instance manager of the web sdk, in that it collects all of the instances of the mParticle SDK?

Reply:
Perhaps adding a comment block above each class would be the way to explain each new class's purpose.